constellation/src/storage/rocks_store.rs

```diff
@@ -35,16 +35,38 @@ fn rocks_opts_base() -> Options {
     let mut opts = Options::default();
     opts.set_level_compaction_dynamic_level_bytes(true);
     opts.create_if_missing(true);
+
+    let mut bopts = rocksdb::BlockBasedOptions::default();
+    bopts.set_cache_index_and_filter_blocks(true);
+    bopts.set_pin_l0_filter_and_index_blocks_in_cache(true);
+    bopts.set_bloom_filter(10.0, false);
+    bopts.set_checksum_type(rocksdb::ChecksumType::XXH3);
+    opts.set_block_based_table_factory(&bopts);
+
+    opts.set_min_level_to_compress(1);
```

Comment:
Constellation has a very steady input of new data, so we know L0 is short-lived. L0 tends to store <= 256 MB data, so not much space savings here, so disabling compression reduces CPU spent when reading from L0.
Could be expanded to L2 maybe.